environment/src/main/java/com/sequenceiq/environment/environment/validation/network/aws/AwsEnvironmentNetworkValidator.java

```diff
@@ -43,22 +43,22 @@ public void validateDuringFlow(EnvironmentDto environmentDto, NetworkDto network
                 return;
             }
             if (networkDto.getSubnetMetas().size() != cloudmetadata.size()) {
-                message = String.format("Subnets of the environment (%s) are not found in the vpc (%s).",
+                message = String.format("Subnets of the environment (%s) are not found in the VPC (%s). All subnets are expected to belong to the same VPC",
```

Comment:
I am sorry but specified the wrong error message in the jira. 
I would write: 
"One or more subnets specified in the environment (%s) were not found in the VPC (%). Please make sure that the submitted subnet ids are spelled correctly and do exist in the VPC."